lib/combinePrimitives.js

```diff
@@ -0,0 +1,270 @@
+'use strict';
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+var defaultValue = Cesium.defaultValue;
+var DeveloperError = Cesium.DeveloperError;
+var loadGltfUris = require('./loadGltfUris');
+module.exports = combinePrimitives;
+
+//Combines primitives in a mesh which have the same material, mode, and attributes.
+function combinePrimitives(gltf) {
+    if (defined(gltf.accessors) && defined(gltf.bufferViews) && defined(gltf.buffers)) {
+        loadGltfUris(gltf);

```

Comment:
Should this just happen once?  For example, we will build pipelines that have several optimizations and we'll just one to load urls once.

For now, users can just be required to call it, and just include it in the main `.js` file for the command-line tool.